content/tutorials/provision-dashboards-and-data-sources.md

```diff
@@ -252,13 +256,14 @@ For more information on how to configure dashboard providers, refer to [Dashboar
    "timezone": "browser",
    "title": "Cluster",
    "version": 0
-}
-```
+   }
+   ```
 
-- Restart Grafana to provision the new dashboard (or wait 10 seconds for Grafana to automatically create the dashboard).
-- In the sidebar, hover the cursor over **Dashboards** (squares) icon, and click **Manage**. The dashboard appears in a **Services** folder.
+1. Restart Grafana to provision the new dashboard or wait 10 seconds for Grafana to automatically create the dashboard.
+1. In the sidebar, hover the cursor over **Dashboards** (squares) icon, and thenclick **Manage**. The dashboard appears in a **Services** folder.
+((I replaced the options.path with C:\Program Files\GrafanaLabs\grafana\public\dashboards in my JSON file, and this did not work for me. I might have the file in the wrong place, but it is the only "dashboards" folder in the grafana directory, so more likely that the path is the wrong format.))
```

Comment:
Could be wrong format. Did you add the `cluster.json` file in the dashboards folder?

Reply:
I was able to get it to work by fiddling with the path format in default.yaml.